javascript/javascript1/week2/smart-ease2.js

```diff
@@ -0,0 +1,135 @@
+//Flight booking fullname function
+
+function getFullname (firstname, surname, useFormalName) {
+    if (useFormalName == true) {
+        return "Lord " + firstname + " " + surname;
```

Comment:
Template literals can be nicely used here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals